src/CMakeLists.txt

```diff
@@ -106,7 +106,7 @@ option(ENABLE_GLES "Use OpenGL ES instead of OpenGL" FALSE)
 mark_as_advanced(ENABLE_GLES)
 if(ENABLE_GLES)
 	find_package(OpenGLES2 REQUIRED)
-elseif()
+elseif(BUILD_CLIENT)
```

Comment:
Just put the entire enable_gles stuff in an `if(BUILD_CLIENT)`

Reply:
Right. Fixing ...

Reply:
Fixed.